tests/phpunit/CRM/Activity/BAO/ActivityTest.php

```diff
@@ -757,23 +757,18 @@ public function testbyActivityDateAndStatus() {
     $lastWeekActivities = array(1, 2, 3);
     $todayActivities = array(4, 5, 6, 7);
     $lastTwoMonthsActivities = array(8, 9, 10, 11);
-    $lastYearActivties = array(12, 13, 14, 15, 16);
```

Comment:
does this mean the unit test works for the 5 years that follow 2012, 2013, 2014, 2015, 2016?

Reply:
This means five activities were having activity date time of last year i.e. ```strtotime('1 year ago')```. Since we are on 1st week and 1st month of this year reason why activity count is 12 (lastWeekActivities + lastTwoMonthsActivities + lastYearActivties) as these activities comes under last year 4rth Jan 2017 - 4rth Jan 2018